projects/storefrontstyles/scss/components/form-components/_star-rating.scss

```diff
@@ -1,19 +1,26 @@
 %cx-star-rating {
-  .star {
-    position: var(--cx-position, relative);
-    display: var(--cx-display, inline-block);
-    color: var(--cx-color, var(--cx-g-color-light));
-    margin: var(--cx-margin, 0 5px 0 0);
-  }
+  padding: 15px 0;
 
-  .full {
-    color: var(--cx-color, var(--cx-g-color-primary));
+  &:not([disabled='true']) {
+    cursor: pointer;
```

Comment:
Also I'm putting this comment here but it's not related: in `_product-summary.scss` class `rating` (line 32) has a bottom margin of 12px. I would remove it with these new stars.